dev-tools/harness.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Random Simulation harness for testing and benchmarking purposes.
+ *
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * @license MIT
+ */
+
+'use strict';
+
+const BattleStreams = require('../.sim-dist/battle-stream');
+const Dex = require('../.sim-dist/dex');
+const PRNG = require('../.sim-dist/prng').PRNG;
+const RandomPlayerAI = require('../.sim-dist/examples/random-player-ai').RandomPlayerAI;
+const Streams = require('../.lib-dist/streams');
+
+const DEFAULT_SEED = [0x09917, 0x06924, 0x0e1c8, 0x06af0];
```

Comment:
I am afraid this is a very boring seed.

IIRC, calling ``.sample(array)`` will first result in picking the first element at least 12 consecutive times for ``.length < 8``. Therefore, the tested formats will be highly biased, as well as the chosen moves.

Though maybe I am worrying over something of no actual relevance.

Reply:
Oh, thanks for letting me know! For my original benchmarking experiments I was using `node --random-seed`, but decided to switch to the `PRNG` for this more 'polished' version. I don't know that the seed matters a ton given I run at default 100 games and usually over 1000, but no reason to choose a known 'bad' seed. Any suggestions? usually something like `0,0,0,0` is bad for a lot of generators.

Reply:
We could get the upper 64 bits of a crypto hash of the date. That would allow for both some consistency and variability in the results.

Reply:
Did you steal the default seed from the tests? The test seed is I think designed to minimize critical hits.

Reply:
We commonly use `1,2,3,4` as a seed. I don't know if that's bad for the generator.